soc/sparc/at697/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+zephyr_compile_options(-mcpu=leon -mtune=leon)
```

Comment:
Find a way to avoid specifying -mcpu=leon multiple times.

Reply:
In the current version, we only have "-mcpu=leon -mtune=leon" only in this file.